edr_server/core/models/extents.py

```diff
@@ -252,7 +252,7 @@ class Extents(EdrModel["Extents"]):
     Based on
     https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/blob/546c338/standard/openapi/schemas/extent.yaml
     """
-    spatial: SpatialExtent
```

Comment:
This allows us to use `from_json` on the Parameter Extent for dimensionless cubes

Reply:
I'd double check the OpenAPI schema for `extent` - I have a feeling that it's a required field (hence why it wasn't already optional). If it is required, then the `to_json` for `Extents` will need to ensure a valid representation of an empty `SpatialExtent` is included when `spatial` is set to `None`

Reply:
I think `spatial` is optional, going off the comment on line 263 - or do you mean `extent` itself?

Reply:
I just looked, the comment is correct, the `extent.yaml` schema has no `required` fields